.changes/unreleased/Fixes-20220523-103843.yaml

```
@@ -0,0 +1,8 @@
+kind: Fixes
+body: Remove the default 256 characters limit on postgres character varying type when
+  no limitation is set
+time: 2022-05-23T10:38:43.392232+02:00
+custom:
+  Author: shrodingers
+  Issue: "636"
```

```suggestion
  Issue: "5238"
```
This should be the GitHub issue number.  636 is the issue number in our Jira instance.